superduperdb/vector_search/inmemory.py

```diff
@@ -20,6 +21,9 @@
     to_numpy,
 )
 
+if t.TYPE_CHECKING:
+    import torch
```

Comment:
And, unfortunately, here too.

Another way would be to distribute just the typing information for `torch` with our code, seems painful.

Or perhaps I'm not right with my fears, but I don't see how they could be avoided...

Reply:
Same as my last comment.